### PR TITLE
feature/exlm-890

### PR DESCRIPTION
**Pull request by adobenitesh**
Jira ID: EXLM-890

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/browse
- After: https://feature-exlm-890--exlm--adobe-experience-league.hlx.live/en/browse

### CODE REVIEW

File: scripts/browse-card/browse-cards-coveo-data-adaptor.js [hunk 1]
```diff
@@ -93,6 +94,7 @@ const BrowseCardsCoveoDataAdaptor = (() => {
       copyLink: url,
       viewLink: url,
       viewLinkText: placeholders[`viewLink${convertToTitleCase(contentType)}`] || 'View',
+      id: parentResult?.el_id || el_id || '',
```
Comment (noormd08) [inline scripts/browse-card/browse-cards-coveo-data-adaptor.js:97]: Please keep the id on the top

File: scripts/data-model/browse-cards-model.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ const browseCardDataModel = {
   bookmarkLink: '',
   viewLink: '',
   viewLinkText: '',
+  id: '',
```
Comment (noormd08) [inline scripts/data-model/browse-cards-model.js:22]: Move it to top

File: scripts/browse-card/browse-cards-coveo-data-adaptor.js (hunk 1 above)
Comment (adobenitesh) [reply scripts/browse-card/browse-cards-coveo-data-adaptor.js]: Done

File: scripts/data-model/browse-cards-model.js (hunk 2 above)
Comment (adobenitesh) [reply scripts/data-model/browse-cards-model.js]: Done

**Review by noormd08 — APPROVED** [review]

**Review by Sivaramvt — APPROVED** [review]

**Pull request merged**